rllib/models/torch/recurrent_net.py

```diff
@@ -10,14 +10,14 @@
 
 
 @DeveloperAPI
-class RecurrentTorchModel(TorchModelV2, nn.Module):
+class RecurrentNetwork(TorchModelV2, nn.Module):
```

Comment:
This change seems fine, but note that it is a breaking DeveloperAPI change.

Reply:
I'll make sure, we have a backward compatibility in place as well as a warning.